### PR TITLE
FIX Cannot link an image in TinyMCE

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Description
We should check that user selected an element and if element doesn't have inner content return outer HTML. 
## Parent issue
- https://github.com/silverstripe/silverstripe-admin/issues/1548

### CODE REVIEW

File: client/src/legacy/TinyMCE_sslink-anchor.js
Comment (sabina-talipova) [inline client/src/legacy/TinyMCE_sslink-anchor.js:1]: I copied changes from CMS 5.

**Comment by sabina-talipova** [comment]
I copied changes from CMS 5.

**Comment by sabina-talipova** [comment]
[This PR](https://github.com/silverstripe/silverstripe-admin/pull/1571) is required to pass test

**Comment by GuySartorelli** [comment]
Superceded by https://github.com/silverstripe/silverstripe-cms/pull/2883

**Pull request closed without merging**